modules/core/ante/ante_test.go

```diff
@@ -45,14 +45,14 @@ func TestAnteTestSuite(t *testing.T) {
 }
 
 // createRecvPacketMessage creates a RecvPacket message for a packet sent from chain A to chain B.
-func (suite *AnteTestSuite) createRecvPacketMessage(sequenceNumber uint64, isRedundant bool) sdk.Msg {
-	packet := channeltypes.NewPacket(ibctesting.MockPacketData, sequenceNumber,
+func (suite *AnteTestSuite) createRecvPacketMessage(isRedundant bool) sdk.Msg {
+	sequence, err := suite.path.EndpointA.SendPacket(clienttypes.NewHeight(2, 0), 0, ibctesting.MockPacketData)
+	suite.Require().NoError(err)
+
+	packet := channeltypes.NewPacket(ibctesting.MockPacketData, sequence,
 		suite.path.EndpointA.ChannelConfig.PortID, suite.path.EndpointA.ChannelID,
 		suite.path.EndpointB.ChannelConfig.PortID, suite.path.EndpointB.ChannelID,
-		clienttypes.NewHeight(1, 0), 0)
-
-	err := suite.path.EndpointA.SendPacket(packet)
-	suite.Require().NoError(err)
+		clienttypes.NewHeight(2, 0), 0)
```

Comment:
was this supposed to change?

Reply:
Good catch. It was not intentional, but I copied from main. I will change it back